project.md

```diff
@@ -5,47 +5,79 @@
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Since my domain is limited to creating a calendar, a DSL is appropriate as it will allow me to create a language of text inputs that can be translated into a visual output of a calendar. This will be a great opportunity for me to practice writing a DSL as I will need to determine how I want inputs to be structured and what types of parameters the inputs can include. It addresses the need because it will allow users to create a calendar based on text input which will hopefully save many people time. My DSL will hopefully serve as a useful tool to automate the tedious process of creating an online calendar one element at a time. The domain is niche yet widely used which is why I believe that a DSL is appropriate and needed.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I'm excited by this idea because it is something that I would actually use and I know other people who would benefit from it too. I came up with the idea after interviewing a CMC environmental science major, who told me about how they rely on their computer for scheduling their work. After thinking about what they said, I thought about how I interact with my computer in the same way and realized that I had a few pain points in this area. Currently, as a Google Calendar user, I have to manually input each event into the calendar which can become tedious when I have multiple similar events (study times) to enter at once. Also, Google calendar doesn't have a great full-month view which I like to use. I am excited to potentially create a solution to these pain points as I will use my project if it is built well. I find it extremely motivating to work on something that I will actually use. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Easy full-semester time management
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+The user would interact with the language by initially creating a calender by setting the start and end dates. Then they would have the ability to program events by specifying the day they want to edit and choosing to either add, remove or alter items that they want to schedule. The calendar would be viewed in months (that's how I view my time management) and would help students visualize and map out their semester. Programming would look like users editing, adding, and checking things off their calendars. The right way to interact with the problem domain would be to input items into the calendar as soon as one knows about them and cross them off once they are complete. Also, I want the user to be able to cross off entire days once they have passed or if there is a holiday ahead that requires no work. My DSL will hopefully be so easy to use that these tasks will take little time to do.
+
```

Comment:
It seems that the syntax of your project will lean a bit more natural language-y. There is a dynamic/automation component in your description (inputs and actions _as soon as_ user do x, “crossing off” items _as soon as_ it is complete) that might not be a core design of your DSL. If you find yourself tackling a lot of work, maybe parts of your syntax that don’t directly relate to initial calendar creation can be a system component you can add as a stretch goal.